modules/analysis/src/config.rs

```diff
@@ -0,0 +1,18 @@
+fn u64_from_size(s: &str) -> Result<u64, parse_size::Error> {
+    // keep it simple, allow only binary units (1024 based)
+    let cfg = parse_size::Config::new().with_binary();
+    cfg.parse_size(s)
+}
+
+#[derive(clap::Args, Debug, Clone)]
+pub struct AnalysisConfig {
+    #[arg(
+        id = "max-cache-size",
+        long,
+        env = "TRUSTD_MAX_CACHE_SIZE",
+        default_value = "200 MB",
+        help = "Maximum size of the graph cache.  You can use binary units like 'kb', 'mb' or 'gb'.",
+        value_parser = u64_from_size
```

Comment:
I'd like to keep this aligned with the existing way to parse byte sizes: https://github.com/trustification/trustify/blob/a7e2855368841cd6e667b35aab5608d411330749/common/infrastructure/src/app/http.rs#L134-L141

Reply:
will do